src/main/java/org/apache/fineract/api/BatchApi.java

```diff
@@ -21,9 +26,11 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Optional;
-
+import java.util.ArrayList;
+import static org.apache.fineract.core.service.OperatorUtils.dateFormat;
```

Comment:
@danishjamal104 Static imports are usually at the top. Checkstyle your code.

Reply:
FIXED